l1-contracts/contracts/upgrades/BaseZkSyncUpgrade.sol

```diff
@@ -203,10 +206,9 @@ abstract contract BaseZkSyncUpgrade is ZkSyncHyperchainBase {
 
         // We want the hashes of l2 system upgrade transactions to be unique.
         // This is why we require that the `nonce` field is unique to each upgrade.
-        require(
-            _l2ProtocolUpgradeTx.nonce == _newProtocolVersion,
-            "The new protocol version should be included in the L2 system upgrade tx"
-        );
+        if (_l2ProtocolUpgradeTx.nonce != _newProtocolVersion) {
+            revert NewProtocolVersionNotInUpgradeTxn();
```

Comment:
What is Upgrde Tx**n**? Txn == transaction? Usually people use tx, but no strong opinion just for my understanding